pages/community.js

```diff
@@ -17,7 +17,7 @@ class CommunityPage extends Component {
     const { router: { query }, seed } = this.props
 
     if (query.share) {
-      return <Share focus={query.share} first={1} />
+      return <Share focus={query.share} pkg={query.pkg} />
```

Comment:
Currently, https://gen.republik.black/community?share=true&pkg=PROLONG shows a 500

```
12:20:36 AM republik.1  |  graphql error in undefined (undefined): { error: column "slug" does not exist
12:20:36 AM republik.1  |      at Connection.parseE (/Users/amoor/dev/republik/backends/node_modules/pg/lib/connection.js:546:11)
12:20:36 AM republik.1  |      at Connection.parseMessage (/Users/amoor/dev/republik/backends/node_modules/pg/lib/connection.js:371:19)
12:20:36 AM republik.1  |      at Socket.<anonymous> (/Users/amoor/dev/republik/backends/node_modules/pg/lib/connection.js:114:22)
12:20:36 AM republik.1  |      at Socket.emit (events.js:182:13)
12:20:36 AM republik.1  |      at Socket.EventEmitter.emit (domain.js:442:20)
12:20:36 AM republik.1  |      at addChunk (_stream_readable.js:283:12)
12:20:36 AM republik.1  |      at readableAddChunk (_stream_readable.js:264:11)
12:20:36 AM republik.1  |      at Socket.Readable.push (_stream_readable.js:219:10)
12:20:36 AM republik.1  |      at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
12:20:36 AM republik.1  |    message: 'column "slug" does not exist',
12:20:36 AM republik.1  |    locations: [ { line: 2, column: 3 } ],
12:20:36 AM republik.1  |    path: [ 'statements' ] }
```

Reply:
Fixed the various bugs in FE and BE. But we always use `share` with an `userId`. Which already worked.